src/OpenTelemetry/Metrics/Builder/MeterProviderBuilderExtensions.cs

```diff
@@ -319,36 +319,54 @@ public static MeterProvider Build(this MeterProviderBuilder meterProviderBuilder
 
 #if EXPOSE_EXPERIMENTAL_FEATURES
     /// <summary>
-    /// Sets the <see cref="ExemplarFilter"/> to be used for this provider.
-    /// This is applied to all the metrics from this provider.
+    /// Sets the <see cref="ExemplarFilterType"/> to be used for this provider
+    /// which controls how measurements will be offered to exemplar reservoirs.
+    /// Default provider configuration: <see
+    /// cref="ExemplarFilterType.AlwaysOff"/>.
```

Comment:
The default is `TraceBased`

Reply:
need some spec work to clarify defaults. It is not clear if exemplars, as a feature, itself is enabled by default. once enabled, yes, tracebased is the default, but we need to see if that makes sense. (I don't think so, but will wait for Blanch's current work before we approach spec)

Reply:
Agree it is a bit confusing in the spec.

There's two statements in [metrics sdk](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md):

> Exemplar sampling SHOULD be turned off by default. If Exemplar sampling is off, the SDK MUST NOT have overhead related to exemplar sampling.

> The ExemplarFilter SHOULD be a configuration parameter of a MeterProvider for an SDK. The default value SHOULD be TraceBased.

I tried to satisfy that 🤣